app/models/energy/programtype.py

```diff
@@ -41,6 +49,12 @@ class PeopleAbridged(BaseModel):
         description='Used to specify a fixed latent fraction of heat gain due to people.'
     )
 
+    @validator('latent_fraction')
+    def check_string_latent_fraction(cls, v):
+        if not isinstance(v, float) and v != 'autocalculate':
+            raise ValueError( 'This is not a valid entry for latent_fraction')
```

Comment:
Maybe you should do something like this instead of using the word "This":
```
raise ValueError( '"{}" is not a valid entry for latent_fraction'.format(v))
```
That error message will be a little more helpful to people since they will see what their incorrect input was.